vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.cc

```diff
@@ -49,6 +50,7 @@ std::string MediaTwitch::GetMediaIdFromParts(
         std::string id("");
```

Comment:
let's move this one to the top of the function and we can then return just once

Reply:
also we can just do std::string id;

Reply:
Done